README.md

```diff
@@ -307,6 +307,18 @@ ckan
 
 ```
 
+## Mock harvest source
+
+The docker compose configurations start an instance of the static [mock harvest source](https://github.com/alphagov/ckan-mock-harvest-sources)
+which should make it easy to populate the database with some basic standard content. To use this, add a harvest
+source with the address `http://static-mock-harvest-source:11088` and of type "CKAN". The content served by this
```

Comment:
I don't think the port is needed here for the harvest source

Reply:
It is because if you then try and access a harvested package from a browser and expect to be able to follow the link, without the port your browser will try to connect to your localhost on port 80, which won't work.

Reply:
It may _appear_ to work initially without the port, though.

Reply:
Without the port I can get a harvest going and have a fake dataset.

Reply:
Yes but when you follow the link from within the dataset page, it will link back to localhost (port 80), and that link will not work.

Reply:
I switched branch on ckan-mock-harvest-source and it is working now